org.eclipse.xtext.extras.tests/src-gen/org/eclipse/xtext/generator/grammarAccess/services/GrammarAccessTestLanguageGrammarAccess.java

```diff
@@ -64,16 +64,16 @@
 		private final Action cATypeAction_1 = (Action)cGroup.eContents().get(1);
 		
 		//AType root::AType:
-		//	'foo' {root::AType};
+		//	'foo' {root :: AType};
```

Comment:
@ArneDeutsch Missing formatting rule for qualified types in Actions? I wouldn't expect a space around `::`

Reply:
problem is indeed still there

Reply:
Tested and fixed.